File: latihan3 b/Latihan3 PHP/Processor.php
<?php

class Processor
{ 
	/*kelas yang digunakan untuk mengimplementasikan Processor/
	/*atribut Processor*/
		
		private $name = "";		//name
		private $price = 0;		//price
	
		public function __construct($name ="", $price = 0){
		/*costructor*/
		
			$this->name = $name;
			$this->price = $price;
		}
	
	/*mengeset nilai name*/
	public function setname($name)
	{
		$this->name = $name;
	}
	
	/*mengeset nilai price*/
	public function setprice($price)
	{
		$this->price = $price;
	}
	
	/*mengembalikan nilai name*/
	public function getname()
	{
		return $this->name;
	}
	
	/*mengembalikan nilai price*/
	public function getprice()
	{
		return $this->price;
	}
	
	function __destruct(){
		/*desstructor*/
		}
}
?>

File: latihan3 b/Latihan3 Py/Processor.py
class Processor():
#kelas yang digunakan untuk mengimplementasikan Processor
#atribut Processor
	

	name = ""		#name
	price = 0		#price
	
	def __init__(self, name="", price = 0):
	#costructor
		self.name = name
		self.price = price
        
    #mengeset nilai name
	def setname(self, name):
		self.name = name
	
	#mengeset nilai price
	def setprice(self, price):
		self.price = price
        
    #mengembalikan nilai name
	def getname(self):
		return self.name   

	#mengembalikan nilai price
	def getprice(self):
		return self.price

	
	